assets/stylesheets/main.scss

```diff
@@ -1,12 +1,9 @@
-
-$cg-blue: #1f2c38;
-
 .usa-alert-all-passing {
   background-image: none;
 }
 
 .usa-disclaimer {
```

Comment:
In this main.scss file a custom CSS file on compliance viewer to override things from WDS and cg-style?